src/content/docs/tips/astro-font.mdx

```diff
@@ -0,0 +1,160 @@
+---
+title: Setup Astro-Font
+description: Quickly configure Astro-Font for your Astro project
+astroRange: ^4.0.0
+---
+
+import { Tabs, TabItem, Steps } from "@astrojs/starlight/components";
+
+Install `astro-font` package using your preferred package manager:
+
+<Tabs>
+    <TabItem label="npm">
+        ```sh
+        npm install astro-font
+        ```
+    </TabItem>
+    <TabItem label="pnpm">
+        ```sh
+        pnpm add astro-font
+        ```
+    </TabItem>
+    <TabItem label="yarn">
+        ```sh
+        yarn add astro-font
+        ```
+    </TabItem>
+</Tabs>
+
+## With Local Fonts
+
+<Steps>
+1. Make sure you have the local font in `public/fonts` directory. This example will use IBM Plex Sans
+  
+2. Configure `astro-font` in `layout.astro`
+
+   ```astro title="layout.astro"
+   ---
+   import { AstroFont } from "astro-font";
+   ---
+   <AstroFont
+      config={[
+        {
+          name: "IBMPlexSans",
+          src: [
+            {
+              style: 'normal',
+              weight: '400',
+              path: './public/fonts/IBMPlexSans-Regular.ttf'
+            },
+          ],
+          preload: true,
+          display: "swap",
+          fallback: "sans-serif",
+          cssVariable: "font-ibm"
+        }
+      ]}
+    />
+   ```
+
+3. Start using the font
+  
+   ```css title="globals.css"
+   body {
+       font-family: "font-ibm"
+   }
+   ```
+  
+</Steps>
+
+## With Google Fonts
+
+<Steps> 
+1. Configure `astro-font` in `layout.astro`. This example will use Inter font
+
+   ```astro title="layout.astro"
+   ---
+   import { AstroFont } from "astro-font";
+   ---
+   <AstroFont
+      config={[
+        {
+          src: [],
+          name: "Inter",
+          preload: false,
+          display: "swap",
+          fallback: "sans-serif",
+          cssVariable: "font-inter",
+          googleFontsURL: 'https://fonts.googleapis.com/css?family=Inter',
+        },
+      ]}
+    />
+   ```
+
+2. Start using the font
+  
+   ```css title="globals.css"
+   body {
+       font-family: "font-inter"
+   }
+   ```
+  
+</Steps>
+
+## With TailwindCSS
```

Comment:
I think we can just reference the official docs here https://docs.astro.build/en/guides/fonts/#register-fonts-in-tailwind